app/lib/surface/LayerAdapter.coffee

```diff
@@ -263,10 +264,10 @@ module.exports = LayerAdapter = class LayerAdapter extends CocoClass
       actionNames = (bundle.actionName for bundle in bundleGrouping)
       args = [thangType, colorConfig, actionNames, builder]
       if thangType.get('raw') or thangType.get('prerenderedSpriteSheetData')
-        if (thangType.get('spriteType') or @defaultSpriteType) is 'segmented'
-          @renderSegmentedThangType(args...)
-        else
-          @renderSingularThangType(args...)
+        switch (thangType.get('spriteType') or @defaultSpriteType)
+          when 'segmented' then @renderSegmentedThangType(args...)
+          when 'animate' then throw new Error('animate sprite type not supported')
```

Comment:
This code path runs and throws errors everywhere. (CInematics, Level Editor, and play view).

This is not a big  problem as this is not going to be seen by users yet.

While testing I noticed that we don't seem to be able to display multiple instances of the animate sprite type.
To repro. Import some art under the `animate` sprite type.
Change the `kind` property to `misc`.
Then make a new level and click and drop the thang into the level. I can only make 1 appear.